data/json/npcs/classes.json

```diff
@@ -421,5 +421,50 @@
             [ "FLEET2", 100 ],
             [ "ANTENNAE", 100 ]
         ]
+    },{
+        "type" : "npc_class",
+        "id" : "NC_SLIME",
+        "name" : "Slimeling",
+        "job_description" : "I'm you!",
+        "common" : false,
+        "bonus_int" : -100,
+        "worn_override": "EMPTY_GROUP",
+        "carry_override": "EMPTY_GROUP",
+        "weapon_override": "EMPTY_GROUP",
+        "traits": [
+            [ "THRESH_SLIME", 100 ],
+            [ "POISRESIST", 100 ],
+            [ "CHEMIMBALANCE", 100 ],
+            [ "REGEN", 100 ],
+            [ "RADIOGENIC", 100 ],
+            [ "DISIMMUNE", 100 ],
+            [ "PARAIMMUNE", 100 ],
+            [ "POISONOUS", 100 ],
+            [ "SLIMY", 100 ],
+            [ "SLIME_HANDS", 100 ],
+            [ "DEX_UP", 100 ],
+            [ "INT_UP", 100 ],
+            [ "SMELLY2", 100 ],
+            [ "DEFORMED3", 100 ],
+            [ "HOLLOW_BONES", 100 ],
```

Comment:
Hollow bones will make them go fast.

Reply:
Is there a particular reason they shouldn't go fast? They will be able to attack faster, however this is balanced out by their frailty. If the concern is them running ahead of or away from the player into danger, the Slimeling's increased movement speed does not appear to be detrimental in that regard; they can be ordered not to fight and the player can still reach enemies first with the Slimeling's following a few spaces behind.